.editorconfig

```diff
@@ -3,20 +3,29 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
+max_line_length = 120
 indent_size = 2
 indent_style = space
 insert_final_newline = true
-max_line_length = 120
 trim_trailing_whitespace = true
 
 [Makefile]
 indent_size = 4
 indent_style = tab
 
+[*.html]
+max_line_length = 1000
+
+[*.js]
+indent_size = 2
+
 [*.md]
-indent_size = 4
 max_line_length = 1000
+indent_size = 4
```

Comment:
_Normally_, but rendering of lists and their items is a bit wild as some differ in placement of list items. Using four is therefore easier as it will entirely prevent that ambiguity.
I think I've seen this issue in some other cases as well with nested elements but I don't remember fully.

And preformatted is also four, which should be one level, else those float with double indentation levels.